common/src/main/java/com/wynntils/core/consumers/functions/FunctionManager.java

```diff
@@ -444,6 +444,9 @@ private void registerAllFunctions() {
         registerFunction(new CombatFunctions.AreaDamageAverageFunction());
         registerFunction(new CombatFunctions.AreaDamagePerSecondFunction());
         registerFunction(new CombatFunctions.BlocksAboveGroundFunction());
+        registerFunction(new CombatFunctions.RepeatedSpellNameFunction());
+        registerFunction(new CombatFunctions.RepeatedSpellCountFunction());
+        registerFunction(new CombatFunctions.RepeatedSpellTicksFunction());
```

Comment:
These are kind of weird names IMO. How about `LastSpellName`, `LastSpellRepeatCount` and `TicksSinceLastSpell`?

Reply:
I thought about those and they were what I chose at first, but LastSpellName implies that it would work all the time, even when a spell "burst" is not in progress. Not sure if this is behaviour that you want, but I kept the scope here purely to spell "bursts"

Reply:
We could make it generic, no? So it handles both bursts and not bursts